packages/story-editor/src/components/floatingMenu/menu.js

```diff
@@ -29,82 +29,85 @@ import {
-        <ContextMenuComponents.MenuSeparator />
-        <ContextMenuComponents.MenuButton>
-          <ContextMenuComponents.MenuIcon
-            title="Dismiss menu"
-            placement={TOOLTIP_PLACEMENT.BOTTOM}
-          >
-            <Icons.Cross />
-          </ContextMenuComponents.MenuIcon>
-        </ContextMenuComponents.MenuButton>
-      </ContextMenu>
-    </MenuWrapper>
-  );
-});
+    return (
+      <MenuWrapper ref={ref} aria-label={__('Design menu', 'web-stories')}>
+        <ContextMenu
+          isInline
+          isHorizontal
+          isSecondary
+          isAlwaysVisible
+          disableControlledTabNavigation
+          aria-label={__('Design options for selected element', 'web-stories')}
+          onMouseDown={(e) => {
+            // Stop the event from bubbling if the user clicks in between buttons.
+            // This prevents the selected element in the canvas from losing focus.
+            e.stopPropagation();
+          }}
+        >
+          <ContextMenuComponents.MenuButton onClick={() => {}}>
+            <ContextMenuComponents.MenuIcon
+              title="Exclaim!"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Icons.ExclamationOutline />
+            </ContextMenuComponents.MenuIcon>
+          </ContextMenuComponents.MenuButton>
+          <ContextMenuComponents.MenuButton onClick={() => {}}>
+            <ContextMenuComponents.MenuIcon
+              title="Bucket!"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Icons.ColorBucket />
+            </ContextMenuComponents.MenuIcon>
+          </ContextMenuComponents.MenuButton>
+          <ContextMenuComponents.MenuButton onClick={() => {}}>
+            <ContextMenuComponents.MenuIcon
+              title="Letter Arrow!"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Icons.LetterTArrow />
+            </ContextMenuComponents.MenuIcon>
+          </ContextMenuComponents.MenuButton>
+          <ContextMenuComponents.MenuSeparator />
+          <ContextMenuComponents.MenuButton onClick={() => {}}>
+            <ContextMenuComponents.MenuIcon
+              title="Pipette!"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Icons.Pipette />
+            </ContextMenuComponents.MenuIcon>
+          </ContextMenuComponents.MenuButton>
+          <ContextMenuComponents.MenuSeparator />
+          <ContextMenuComponents.MenuButton onClick={dismiss}>
+            <ContextMenuComponents.MenuIcon
+              title="Dismiss menu"
+              placement={TOOLTIP_PLACEMENT.BOTTOM}
+            >
+              <Icons.Cross />
+            </ContextMenuComponents.MenuIcon>
+          </ContextMenuComponents.MenuButton>
+        </ContextMenu>
+      </MenuWrapper>
+    );
+  })
+);
 
 FloatingMenu.propTypes = {
-  isVisible: PropTypes.bool,
+  dismiss: PropTypes.func.isRequired,
```

Comment:
Nit: maybe rename to `onDismiss` to make it sound more like a function?

Reply:
True, but somehow it also sounds like a boolean to me currently 🤷‍♀️ Anyway, ok by me if you both think it's ok.

Reply:
What else would it be with that name - we have naming rules for booleans to avoid the confusion. But sure, we can do that. I'll fix that in the next stage PR.

Reply:
Yep, I might be overthinking it 😛